src/input/src/Input.tsx

```diff
@@ -812,6 +812,7 @@ export default defineComponent({
       inputElRef,
       textareaElRef,
       isCompositing: isComposingRef,
+      clear: handleClear,
```

Comment:
To conform exposed type. It should be `clear: () => handleClear()`. Also, `onClear` shouldn't be called since it's an user reaction.